content/en/docs/concepts/policy/node-resources-managers.md

```diff
@@ -0,0 +1,22 @@
+---
+reviewers:
+- derekwaynecarr
+- klueska
+title: Node Resources Managers 
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+In order to support latency-critical and high-throughput workloads, Kubernetes offers a suite of Resource Managers. The managers aim to co-ordinate and optimise node's resources alignment for pods configured with a specific requirement for CPUs, devices, and memory (hugepages) resources. 
+
+<!-- body -->
+
+The main manager, the Topology Manager, is a Kubelet component that co-ordinates the overall resource management process through its [policy](/docs/tasks/administer-cluster/topology-manager/).
```

Comment:
Is it helpful to make it clear that this management always happens _after_ scheduling?

Reply:
I would left it as it's now. There was work started, which relates to NUMA-aware scheduling (do not know what's the status), and this feature could be also discussed in the context of scheduling. We would need to coordinate with people involved.